app/controllers/adventures_controller.rb

```diff
@@ -4,16 +4,46 @@ class AdventuresController < ApplicationController
 
   def index
     @adventure = @adventures.published.first
-    @total = @introduction.adventure_groups.for_index
+    if @adventure.adventure_missions.present?
+      set_mission_variables(@adventure)
+    end
+  end
+
+  def highscore
+    @tr_classes = ['adv-first', 'adv-second', 'adv-third']
```

Comment:
What's this, and why is it here?

Reply:
Classes for first, second and third place on leaderboard but they should probably be defined in the view right?

Reply:
Yep